app/components/curriculum-inventory/verification-preview.hbs

```diff
@@ -12,145 +12,149 @@
       {{did-insert this.setScrollTarget "toc"}}
     >
       <li>
-        <span class="link clickable" role="button" {{on "click" (fn this.scrollTo "table1")}}>
+        <button class="link link-button" type="button" {{on "click" (fn this.scrollTo "table1")}}>
```

Comment:
Not sure these need both `link` and `link-button`, I think maybe just `link-button` would do. Otherwise it's kind of confusing, but I'm also ok with both if it makes this changes smaller.

Reply:
i see your point.
but we actually need both styles here to make that button fully appear like a link - `link-button` gets us there mostly, but not completely. the `link` class adds the rest of the styles to make it look like a link. 
what it boils down is that we need to rename `link-button` to something like `inline-button` and replace it everywhere, and then redefine `link-button` as something that actually lives up to its name.
that's another push that needs to come out of common, we're too far along here in frontend to change that now.